diff --git a/node_modules/react-native-pager-view/android/build.gradle b/node_modules/react-native-pager-view/android/build.gradle
index b46c366..8554e7b 100644
--- a/node_modules/react-native-pager-view/android/build.gradle
+++ b/node_modules/react-native-pager-view/android/build.gradle
@@ -226,7 +226,7 @@ dependencies {
     //noinspection GradleDynamicVersion
   api "com.facebook.react:react-native:+"
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-  implementation 'androidx.viewpager2:viewpager2:1.1.0'
+  implementation 'androidx.viewpager2:viewpager2:1.0.0'
 }
 
 if (isNewArchitectureEnabled()) {